README.md

```diff
@@ -584,7 +584,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Honey Patel](https://github.com/honeykpatel)
   - [Hrithik Chauhan](https://github.com/Hrithik5)
   - [Hulya Karakaya](https://github.com/hulyak)
-
+  - [Hammad Azam](https://github.com/hammadhz)
```

Comment:
Please add your name in alphabetical order

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient

Reply:
ok, I'll check just give me a sec.

Reply:
Take your time :)

Reply:
I made a new pull request. Check out plz let me know if I did it right way.